### PR TITLE
Add Docker definition for nia-monitor

### PR DESCRIPTION
**Pull request by sjmatta**

### CODE REVIEW

**Comment by nathanathan** [comment]
@sjmatta It looks like this needs a SPARQURL environment variable to connect to the DB.

**Comment by sjmatta** [comment]
Good point, I was doing that on the command line but it should be in here.

**Comment by sjmatta** [comment]
@nathanathan That would seem to be environment-dependent, and should be set by the Docker run command.

In [nia-monitor](https://github.com/ecohealthalliance/nia-monitor/blob/10a8e099f565501f27fa867dbdb4e8e56ae94851/imports/startup/server/index.coffee#L1), it defaults to `http://localhost:3030/dataset/query` though, so I can put that in the Dockerfile as a default if you like.

**Comment by sjmatta** [comment]
On the other hand, we do hard-code the environmental variables in [flirt](https://github.com/ecohealthalliance/infrastructure/blob/master/docker/images/flirt/Dockerfile#L75) so I can see that approach being more consistent. However, I don't know what the production setting should be for SPARQURL.

**Comment by nathanathan** [comment]
@sjmatta You can hard-code it to `http://niam.eha.io:3030/dataset/query`

**Comment by sjmatta** [comment]
These are the same as FLIRT but may need updated:
```
ENV MONGO_HOST="192.168.99.100"
ENV MONGO_PORT="27017"
```

File: docker/images/niam/Dockerfile [hunk 1]
```diff
@@ -44,6 +44,7 @@ ENV MONGO_HOST="192.168.99.100"
 ENV MONGO_PORT="27017"
 ENV MONGO_DB="niam"
 ENV MONGO_URI="mongodb://$MONGO_HOST:$MONGO_PORT/$MONGO_DB"
+ENV SPARQURL="http://niam.eha.io:3030/dataset/query"
```
Comment (frosario) [inline docker/images/niam/Dockerfile]: 👍 
This approach works great as opposed to hardcoding it in the actual source code.

Comment (sjmatta) [inline docker/images/niam/Dockerfile]: I was thinking more of hardcoding it in the deploy script, but this is fine with me as well.

**Pull request merged**